components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json

```diff
@@ -515,9 +527,15 @@
     "message": "Wallet Verified",
     "description": "Displayed in wallet that tells users that he is verified"
   },
-  "walletGoToUphold":  {
-    "message": "Go to my Uphold account",
-    "description": "CTA to open uphold account"
+  "walletGoToProvider":  {
+    "message": "Go to my $provider$ account",
+    "description": "CTA to open uphold account",
```

Comment:
Nit: Maybe this description could just say `provider account` instead of `uphold account`.

Reply:
Fixed.